src/dune/upgrader.mli

```diff
@@ -1,4 +1,4 @@
 (** Upgrade projects from jbuilder to Dune *)
 
 (** Upgrade all projects in this file tree *)
-val upgrade : unit -> unit
+val upgrade : ?last:bool -> unit -> unit
```

Comment:
It seems that this param is only used in the recursive call. In order not to pollute the interface, what do you think about extracting a method with a (mandatory) param and leave `upgrade` without param?